docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable releases
+
+The following stable release series are supported for production deployments. Starting with
+version `2.2`, Yugabyte is following a new [release versioning](../versioning) convention.
+The next stable release series is expected to be available in November 2020.
+
+**Note:** Versions `2.1`, `2.0`, and `1.3` that were released before `2.2` are also considered stable
+releases and are supported for existing production deployments.
+
+| Version | Released |
+| :------ | -------- |
+| 2.4                                    | 2020/11 (expected) |
+| [2.2](../earlier-releases/v2.2.0)      | 2020/07/15 |
+| [2.1](../earlier-releases/v2.1.0)      | 2020/02/25 |
+| [2.0](../earlier-releases/v2.0.0)      | 2019/09/17 |
+| [1.3](../earlier-releases/v1.3.0)      | 2019/07/19 |
```

Comment:
@stevebang Need to confirm if 1.3 and 2.0 are still supported ? if so, until when? This is going to be written in our support policy 

*****
  Release Support. 
(a) For the maintenance support period of at least 1 year from the major release date, Yugabyte will provide Updates for such release.

(b) Following the maintenance support period, Yugabyte will provide Updates for at least an additional 180 days subject to the following guidelines:
Updates and Upgrades will not be made to the major release.
Yugabyte will direct Customers to existing Updates and workarounds applicable to the reported case.
Yugabyte may direct Customers to Upgrade to a current release if a workaround does not exist.

(c)  Yugabyte will post publicly on its website a notice of End of Life (EOL) for the affected Software and the timeline for discontinuing Support Services. Yugabyte has no obligation to provide Support Services for Software after EOL.

Current Supported Software Releases are listed on Yugabyte’s website.
*****

Reply:
Based on the policy statement above, both of those are still supported. 1.3 would fall under the post-maintenance support time period. I'll keep my eye out for a link to the support policy and add that when it is available.